src/lib/components/library/badge.svelte

```diff
@@ -0,0 +1,9 @@
+<span class="badge rounded-lg bg-black px-3 py-[1.5px]">
+	<slot />
+</span>
+
+<style lang="postcss">
+	.badge {
+		color: var(--grey-slight);
```

Comment:
Here for example I just need that custom css, because there's no config for this color in that scenario